spec/models/user_spec.rb

```diff
@@ -111,7 +111,7 @@
       let(:rolled_over_user) { build(:user, :notifications_configured, associated_with_accredited_body: true, notifications_configured: true) }
 
       it "raises an error when trying to change state from 'rolled_over' to 'subscribed_to_notifications'" do
-        expect { rolled_over_user.accept_notifications_screen! }.to raise_error(AASM::InvalidTransition)
+        expect { rolled_over_user.accept_notifications_screen! }.to_not raise_error
```

Comment:
Same here 
Asserting an error is not raised - test description needs to be updated

Reply:
Good find, changed it!